### PR TITLE
Feature/e UI 6714

### PR DESCRIPTION
**Pull request by SivaprakasamSivaprakasam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6714

### Change description ###

https://tools.hmcts.net/jira/browse/EUI-6714

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by maliumer — APPROVED** [review]

File: src/shared/components/palette/case-link/components/read-linked-cases.component.ts [hunk 1]
```diff
@@ -68,9 +68,8 @@ export class ReadLinkedCasesComponent implements OnInit, AfterViewInit {
   }
 
   public reloadCurrentRoute() {
-    let currentUrl = this.router.url;
-    this.router.navigateByUrl('/', { skipLocationChange: true }).then(() => {
-      this.router.navigate([currentUrl]);
+    this.router.navigate(['cases', 'case-details', this.linkedCasesService.caseDetails.case_id]).then(() => {
+      window.location.hash = 'Linked cases';
```
Comment (rayliangatos) [inline src/shared/components/palette/case-link/components/read-linked-cases.component.ts:72]: 1. suppose it shouldn't use blank string in the hash? 
2. and does it have that hash key in the html template?

File: src/shared/components/palette/case-link/components/before-you-start/before-you-start.component.ts [hunk 2]
```diff
@@ -35,4 +34,10 @@ export class BeforeYouStartComponent {
       navigateToNextPage: true
     });
   }
+
+  public onBack(): void {
+    this.router.navigate(['cases', 'case-details', this.linkedCasesService.caseId]).then(() => {
+      window.location.hash = 'Linked cases';
```
Comment (rayliangatos) [inline src/shared/components/palette/case-link/components/before-you-start/before-you-start.component.ts:40]: 1. suppose it shouldn't use blank string in the hash? 
2. and does it have that hash key in the html template?

Comment (SivaprakasamSivaprakasam) [reply src/shared/components/palette/case-link/components/before-you-start/before-you-start.component.ts:40]: As far as I know, there is no hash key for the linked  cases and we usually redirect to the landing page tab.

File: src/shared/components/palette/case-link/components/read-linked-cases.component.ts (hunk 1 above)
Comment (SivaprakasamSivaprakasam) [reply src/shared/components/palette/case-link/components/read-linked-cases.component.ts:72]: As far as I know, there is no hash key for the linked  cases and we usually redirect to the landing page tab.

Comment (rayliangatos) [reply src/shared/components/palette/case-link/components/read-linked-cases.component.ts:72]: so does that mean this window.location.hash never work can we remove it?

File: src/shared/components/palette/case-link/components/before-you-start/before-you-start.component.ts (hunk 2 above)
Comment (rayliangatos) [reply src/shared/components/palette/case-link/components/before-you-start/before-you-start.component.ts:40]: so does that mean this window.location.hash never work can we remove it?

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**